.github/workflows/changelogs/changelogs.py

```diff
@@ -31,6 +31,7 @@ class RegexRules:
     WRONG_CAP_START = re.compile(r"^\W*[a-z]")
     WRONG_CAP_AFTER = re.compile(r"\. *[a-z]")
     WRONG_SPACING = re.compile(r"([.,;:])[^ \n]")
+    VERSION_REGEX = re.compile(r"\b(?:v?(\d+\.\d+(\.\d+)?)(?:[-\.][a-zA-Z0-9]+)?)\b")
```

Comment:
```suggestion
    VERSION_REGEX = re.compile(r"\b(?:v?(\d+\.\d+(\.\d+)?)(?:[-.][a-zA-Z0-9]+)?)\b")
```
Escaping the dot is not necessary inside `[]`:
```
>>> re.match(r"[.]", ".")
<re.Match object; span=(0, 1), match='.'>
>>> print(re.match(r"[.]", "x"))
None
```